packages/css/src/components/card/readme.md

```diff
@@ -1,3 +1,3 @@
 # `.dso-card`
 
-Er dient door de implementator doormiddel van script een doorklik gemaakt te worden op de `.dso-card`
+Er dient door de implementator een click event op `.dso-card` te worden gezet. Hiermee kan de functionaliteit van het doorklikken worden gebouwd.
```

Comment:
"Dit is een HTML/CSS component, de dso-toolkit biedt dus geen klik-gedrag aan. Hiervoor zou het webcomponent ingezet kunnen worden `<dso-card>`. Bij gebruik van het HTML/CSS component zijn afnemers zelf verantwoordelijk voor het plaatsen van een click eventhandler op `.dso-card`"

Reply:
Tekst aangepast.